rskj-core/src/main/resources/config/rsk-sample.conf

```diff
@@ -92,6 +88,15 @@ peer {
     maxActivePeers = 30
 }
 
+# Interface to bind peer discovery and wire protocol
+# Make sure you are using the correct bind ip. Wildcard value: 0.0.0.0
+bind.address = 0.0.0.0
+
+# public IP/hostname which is reported as our host during discovery
+# if not set, the service http://checkip.amazonaws.com is used
+# the last resort is to get the bind.ip address
```

Comment:
The comment should says `# the last resort is to get the public.ip address` instead of bind.ip